internal/cmd/server.go

```diff
@@ -37,7 +38,7 @@ func GetServerCommand() *cobra.Command {
 		Aliases: []string{"serve"},
 		Short:   "Serve Envoy Gateway",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return server(cmd.Context())
+			return server(cmd.Context(), cmd.OutOrStdout())
```

Comment:
note: this is the top level change that allows me to configure the log output by cmd.SetOut